docs/core/tools/dotnet-tool-install.md

```diff
@@ -116,9 +110,7 @@ For more information, see [Install a local tool](global-tools.md#install-a-local
 
   Specifies the location where to install the Global Tool. PATH can be absolute or relative. If PATH doesn't exist, the command tries to create it. Omitting both `--global` and `--tool-path` specifies a local tool installation.
 
-- **`-v|--verbosity <LEVEL>`**
-
-  Sets the verbosity level of the command. Allowed values are `q[uiet]`, `m[inimal]`, `n[ormal]`, `d[etailed]`, and `diag[nostic]`.
+[!INCLUDE [verbosity](../../../includes/cli-verbosity.md)]
```

Comment:
For these, what's the default verbosity? If it's minimal, consider deleting *cli-verbosity.md* and only keep *cli-verbosity-minimal.md*.

Reply:
Good idea, thanks for the suggestion, but for this and some other commands, the default verbosity wasn't indicated, so I can't be sure it's `minimal`. It seemed safest to leave the default unstated until I get a chance to verify it, at which point I can switch to *cli-verbosity-minimal.md*  or make a new one for a different default. Eventually when all commands' default settings are established I'll be able to delete *cli-verbosity.md*.